### ISSUE TITLE
Support read only fields

### ISSUE THREAD

**Issue by jturkel** [ruby-grape/grape-swagger-entity#17]
The Swagger document format supports marking fields as [read only](http://swagger.io/specification/#definitionsObject). It would be awesome if grape-swagger-entity supported this too e.g.

```ruby
class Person < Grape::Entity
  expose :birthday, documentation: { type: 'date', required: true }
  expose :age, documentation: { type: 'integer', read_only: true }
end
```

**Comment by Bugagazavr** [comment]
thank you, i think we can implement this, right? @LeFnord

**Comment by LeFnord** [comment]
@Bugagazavr … yeap it is very welcome
@jturkel please make a PR, which implements this feature 😄

**Comment by Bugagazavr** [comment]
implemented in 0.2.1.
